lib/util/http/github.ts

```diff
@@ -172,6 +175,66 @@ function constructAcceptString(input?: any): string {
   return acceptStrings.join(', ');
 }
 
+function getGraphqlPageSize(
+  baseUrl: string,
+  fieldName: string,
+  initialCount = 100
+): number {
+  // istanbul ignore if
+  if (ensureTrailingSlash(baseUrl) !== ensureTrailingSlash(githubBaseUrl)) {
```

Comment:
Can you remind me what this is guarding against?

Reply:
It's meant to restrict the feature only to be enabled for GitHub site. However, seems like I've decided it out of doubt and probably was wrong.

Reply:
Removed this guard check, hopefully it would work across all GitHub lineage